File: daily/maxCount.cpp
#include <vector>
#include <iostream>
using namespace std;

class Solution {
    public:
        int maximumCount(vector<int>& nums) {
            int neg = 0, pos = 0;
            for (int n : nums) {
                if (n < 0) neg++;
                else if (n > 0) pos++;
            }
            return max(neg, pos);
        }
    };

int main() {
    Solution sol;
    vector<int> nums = {1, -2, -3, 4};
    cout << sol.maximumCount(nums) << endl;
    return 0;
}

File: daily/maxCount.py
class Solution(object):
    def maximumCount(self, nums):
        pos = 0
        neg = 0
        for n in nums:
            if n > 0: pos += 1
            elif n < 0: neg += 1
        return max(pos, neg)
    
maxCount = Solution().maximumCount
print(maxCount([1, -2, -3, 4])) # 2
print(maxCount([1, 2, 3, 4])) # 4